jsFiles/utility.js

```diff
@@ -86,7 +86,21 @@ function formatTime(date, timeZone = undefined) {
 }
 
 /**
- * handles the alert error messages of fetchData() and geolocation.getCurrentPosition()
+ * handles the alert error of geolocation.getCurrentPosition()
+ * @param {object} error - contains information about the error
+ */
+function handleErrorGeo(error) {
+  if (error.code == 1) {
+    const randomCity = cities[Math.floor(Math.random() * cities.length)];
+    const api = `${apiRootUrl}weather?q=${randomCity}&appid=${KEY}&units=metric`;
```

Comment:
Hm...I'm not sure that it's a good idea to have a side effect inside your errorHandler. 
In general I'll propose to have this fetch data outside of the errorHandler and this function should deal only with what related to the error itself (displaying properly in most of the times).

Reply:
I'm not sure if I understand, I thought this would be the place, since in case of an error (user denying the location) the weather of a random city is to show up instead. The function does deal with error handling otherwise the app well doesn't get any weather's data, like pointed out on the pic of Maria, on this pull request addressed comment. If I were to place this outside this function, then it would execute, and overwrite the fetch from the user location...